.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: php
 sudo: false
 
+## Cache composer bits
+cache:
+  directories:
+    - $HOME/.composer/cache
 php:
-  - 5.5
   - 5.6
   - 7.0
 
 before_script:
-  - cd tests/
+  - composer install --prefer-dist
```

Comment:
Hmmmm I'd say this is working against what travis does.  Unless there's a reason for wanting a particular version of phpunit (e.g. they've added features and travis hasn't upgraded yet) I don't see why we'd do this.